src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -1,21 +1,22 @@
-import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { ActivatedRoute, Params } from '@angular/router';
+import { Component, EventEmitter, Input, OnInit, Output } from "@angular/core";
+import { ActivatedRoute, Params } from "@angular/router";
 
-import { Logger } from '../../shared/logger.service';
-import { WorkItem } from '../work-item';
-import { WorkItemService } from '../work-item.service';
+import { Logger } from "../../shared/logger.service";
+import { WorkItem } from "../work-item";
+import { WorkItemService } from "../work-item.service";
```

Comment:
Why the change of all the single quotes. In JavaScript it is common to use single quotes whenever possible, mostly I think due to the ease of typing it without the Shift key. Additionally, the Angular tutorial uses single quotes. 

This is a change in style that has not been discussed with the team and does not belong in this PR.